### ISSUE TITLE
Finetuning schedule callback

### ISSUE THREAD

**Issue by kirilklein** [FGA-DIKU/EHR#202]
We can speed up fine-tuning, while reaching same performance via a custom FT-schedule, using torch lightning ft [schedule callback](https://lightning.ai/docs/pytorch/stable/notebooks/lightning_examples/finetuning-scheduler.html)
